### PR TITLE
Improve the names of Flutter processes.

### PR DESCRIPTION
**Pull request by pylaligand**

### CODE REVIEW

File: content_handler/app.cc [hunk 1]
```diff
@@ -123,24 +123,35 @@ void App::StartApplication(
...
   if (it == controllers_.end())
     return;
   controllers_.erase(it);
+  UpdateProcessLabel();
+}
+
+void App::UpdateProcessLabel() {
+  std::string label = base_label_;
+  if (controllers_.size() >= 2) {
+    label += " (+" + std::to_string(controllers_.size() - 1) + ")";
```
Comment (abarth) [inline content_handler/app.cc]: If the label would overflow with the added part, I'd truncate the label to MX_MAX_NAME_LEN - suffix.size() - 3 and then append "..." + suffix.  :)

Comment (abarth) [inline content_handler/app.cc]: I'd also do that in the case where controllers_.size() < 2, but obviously suffix would be empty in that case.

**Comment by abarth** [comment]
LGTM

File: content_handler/app.cc [hunk 2]
```diff
@@ -123,24 +123,35 @@ void App::StartApplication(
...
+  }
+  mx::process::self().set_property(MX_PROP_NAME, label.c_str(), label.size());
```
Comment (ianloic) [inline content_handler/app.cc]: The kernel will truncate the name at MX_MAX_NAME_LEN - 1. Maybe there should be logic to truncate the base_label_ rather that losing the number of additional apps?

File: content_handler/app.cc (hunk 1 above)
Comment (ianloic) [inline content_handler/app.cc]: jinks!

Comment (pylaligand) [inline content_handler/app.cc]: Done.

File: content_handler/app.cc (hunk 2 above)
Comment (pylaligand) [inline content_handler/app.cc]: Done.

**Comment by ianloic** [comment]
LGTM. Do you want to add this logic to //apps/dart_runner too? If not I can copy & paste it myself.

**Comment by pylaligand** [comment]
Go for it :)

**Comment by abarth** [comment]
LGTM

**Pull request merged**